src/mui/list/List.js

```diff
@@ -359,7 +358,8 @@ const enhance = compose(
         changeListParams: changeListParamsAction,
         push: pushAction,
     }),
-    translate
+    translate,
+    withPermissionsFilteredChildren
```

Comment:
it seems that you add this HOC always at a different place in the HOC chain. Is there a specific reason? If not, please be consistent.